docs/reference/annotations.rst

```diff
@@ -1,74 +0,0 @@
-Annotations
```

Comment:
Wait... removing this would also end in an 404 in the docs if this url is called.

Can you keep the file and explain that it is not supported anymore and add a link to Marko’s bundle?

It’s more straightforward instead removing it without a note

WDYT?